decidim-core/app/cells/decidim/author_box/show.erb

```diff
@@ -1,10 +1,12 @@
-<div class="<%= author_box_classes %>">
-  <div class="author-data__main">
-    <%= cell "decidim/profile", model %>
-  </div>
-  <% if @options[:extra] %>
-    <div class="author-data__extra">
-      <%= options[:extra] %>
+<div class="card__author">
```

Comment:
Not sure about this. The original layout was extracted from the proposals `show.html.erb` view, which didn't have this class. The M card did have this class, that's why I this class appeared in the `ProposalM` cell.

Can you double check this?